IntelliTect.Multitool/IPaths.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace IntelliTect.Multitool;
+/// <summary>
+/// Provides normalized paths.
+/// </summary>
+public static class IPaths
```

Comment:
As per naming guidelines we should rename this to not have an "I" prefix. To avoid confusion/collisions I would suggest the name "RepositoryPaths"